kymatio/scattering2d/frontend/__init__.py

```diff
@@ -0,0 +1,5 @@
+__all__ = ['Scattering2D_torch', 'Scattering2D_numpy']
+
+
+from .torch_frontend import Scattering2D_torch
```

Comment:
can't really import this if there is no `torch`

Reply:
or maybe yes, if no torch is imported while running through `torch_frontend`, which I doubt

Reply:
we can, because the frontend import the backend. If no backend is loaded or is specified, it should lead to a bug!